src/main/java/com/zufar/onlinestore/payment/service/PaymentEventParser.java

```diff
@@ -0,0 +1,21 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.model.Event;
+import com.stripe.model.EventDataObjectDeserializer;
+import com.stripe.model.PaymentIntent;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class PaymentEventParser {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe

Reply:
Done